edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -56,7 +56,10 @@ impl DockerModuleRuntime {
         // extract base path - the bit that comes after the scheme
         let base_path = get_base_path(docker_url)?;
         let mut configuration = Configuration::new(client);
-        configuration.base_path = base_path.to_str().unwrap().to_string();
+        configuration.base_path = base_path
+            .to_str()
+            .expect("URL path should be percent-encoded ASCII, but was found to be invalid UTF-8")
```

Comment:
Well, yes it's percent-encoded ASCII, but the error is that the *path* that was encoded using percent-encoding turned out to not be valid UTF-8 after it was decoded.

Perhaps something like `"URL points to a path that cannot be represented in UTF-8."`

Reply:
Excellent, I'll update the expect statement